configure

```diff
@@ -1239,6 +1242,11 @@ Installation directories:
 Optional Packages:
   --with-PACKAGE[=ARG]    use PACKAGE [ARG=yes]
   --without-PACKAGE       do not use PACKAGE (same as --with-PACKAGE=no)
+  --with-python-sys-prefix
```

Comment:
I'm curious on where these new options came from and whether we want to document them? It is strange that it looks like the previous `configure` file was generated using 2.71 of autotools but it was not included before. I think I'd prefer not to document these variables by using an `m4` override like we already have for the current help documentation.